### PR TITLE
Added maximum_contiguous_subsequence_sum.c

### PR DESCRIPTION
**Pull request by YashovardhanSiramdas**
**Changes:**
Added Maximum Contiguous Subsequence Sum problem in C which uses Divide and Conquer strategy.

### CODE REVIEW

File: code/divide_conquer/src/maximum_contiguous_subsequence_sum/maximum_contiguous_subsequence_sum.c [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+	}
+
+	return (max(leftSubProblem, rightSubProblem, leftSum + rightSum));
+}
+
+int 
+main()
+{
+	int n;
+	printf("Enter size of Array:- ");
```
Comment (arnavb) [inline code/divide_conquer/src/maximum_contiguous_subsequence_sum/maximum_contiguous_subsequence_sum.c]: `Enter the size of the array: `

Comment (YashovardhanSiramdas) [reply code/divide_conquer/src/maximum_contiguous_subsequence_sum/maximum_contiguous_subsequence_sum.c]: Made suggested changes !

File: code/divide_conquer/src/maximum_contiguous_subsequence_sum/maximum_contiguous_subsequence_sum.c [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+max(int a, int b, int c)
+{
+	if (a > b)
+		return (a > c ? a : c);
+
+	return (b > c ? b : c);
+}
+
+int 
+maximumContiguousSubsequenceSum(int a[], int beg, int end)
```
Comment (iattempt) [inline code/divide_conquer/src/maximum_contiguous_subsequence_sum/maximum_contiguous_subsequence_sum.c]: add `const` for a[]

File: code/divide_conquer/src/maximum_contiguous_subsequence_sum/maximum_contiguous_subsequence_sum.c [hunk 3]
```diff
@@ -0,0 +1,62 @@
+#include <stdio.h>
+
+int 
+max(int a, int b, int c)
```
Comment (iattempt) [inline code/divide_conquer/src/maximum_contiguous_subsequence_sum/maximum_contiguous_subsequence_sum.c]: also here.

**Comment by YashovardhanSiramdas** [comment]
@iattempt Thanks for suggesting changes !

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

**Pull request merged**